run.py

```diff
@@ -145,16 +146,10 @@ def get_requirements(cls, file="requirements.txt"):
 
 
 def finalize_logging():
-    if os.path.isfile("logs/musicbot.log"):
-        log.info("Moving old musicbot log")
-        try:
-            if os.path.isfile("logs/musicbot.log.last"):
-                os.unlink("logs/musicbot.log.last")
-            os.rename("logs/musicbot.log", "logs/musicbot.log.last")
-        except:
-            pass
+    # added code for handing logs with dates in the name
+    musicbotlog = datetime.now().strftime('logs/musicbot_%Y-%m-%d_%H-%M.log')
```

Comment:
Same as above, it doesn't make sense to check the current time in different places when it's supposed to be the startup time.